packages/components/format-date/src/format-date.stories.ts

```diff
@@ -0,0 +1,130 @@
+import { type Meta, type StoryObj } from '@storybook/web-components';
+import { html } from 'lit';
+import '../register.js';
+import { type FormatDate } from './format-date.js';
+
+type Props = Pick<
+  FormatDate,
+  | 'locale'
+  | 'date'
+  | 'year'
+  | 'month'
+  | 'day'
+  | 'weekday'
+  | 'dayPeriod'
+  | 'hour'
+  | 'minute'
+  | 'second'
+  | 'timeZoneName'
+  | 'timeZone'
+  | 'hour12'
+  | 'era'
+> & { fallback?: string };
+type Story = StoryObj<Props>;
+
+export default {
+  title: 'Utilities/Format date',
+  tags: ['draft'],
+  args: {
+    fallback: 'invalid date'
+  },
+  argTypes: {
+    locale: {
+      control: 'inline-radio',
+      options: ['de', 'en', 'es', 'fi', 'it', 'nl', 'no', 'pl', 'sv']
```

Comment:
it says here `en` but that uses the US notation, not the UK one. Maybe we should give that one as an example?